### ISSUE TITLE
[V1] UI wireframes

### ISSUE THREAD

**Issue by chongho** [chungha/q-box#4]
![q-box-v1-wireframes-r1](https://cloud.githubusercontent.com/assets/1796742/19618263/f33b9922-97f9-11e6-973c-0e5ceab0d096.png)

**Comment by chongho** [comment]
Please keep in mind this is a wireframe flow so none of the details (assets / color) are for placement only.

While working on this I came to have questions about feature scope. Hope we can have a discussion around the followings:
- "I don't know" option necessary? (I'd personally like to have this)
- How do you calculate the score (percentage? absolute score?)
- Any other items you want to discuss

**Comment by David-Cho** [comment]
I think that state 3 is very natural to normal test takers.  There might be test takers who don't want to know whether their answers are right or not until the end of the test. After a set of test, they might just want State 5. 

And I love "I don't know" option. It is strongly recommneded. By the way, what if a test taker gets some hints about the quiz instead of seeing the right answer when he or she presses " I don't know" ? 

I prefer absolute score. [If a test maker gives a each question as much points as he wants to set, what do you think about it? ]

**Comment by chungha** [comment]
For the consideration of test takers who want to know the answers later, let's make a new issue with lower priority. We have to keep V1 scope as small as possible for the first small success.

Strongly +1 for "I don't know".

Having each problems variant scores would be a good feature. Let's make an another issue as well.

**Comment by chongho** [comment]
Thank you for the feedback, David and Chungha!

--- Regarding checking out the right answers later, I think it's better to get the right answer right away for these reasons:

• Users get easily distracted and less patient when using mobile apps. There are a lot of real-world interruptions (from other people, cars, noises etc.) so it's important to keep them engaged. Showing visual/sound feedback to them to keep them engaged is important.

• Quite a few successful edu games / quiz apps (like Duolingo, Tinycard etc.) are using the same strategy to make learning experience more light-weight than serious work.

• If we show the right answers at the end of the flow, we need to make a Review answers page that shows all the questions and answers again, which increases the project scope of the current V1&#13;

However I definitely agree that there are different use cases as David mentioned — The flow that feels more like taking an actual test. As Chungha mentioned I agree that we create a separate issue for that.

--- Regarding giving them hints, I like the idea a lot and think we should consider it for V2. I'll create a separate issue for that.

--- Regarding variant scores for each problem, I like it a a lot as well. It'd be good to be included in the V2. I'll create a separate issue for that.

--- Following up on the sync-up with Chungha this morning, I'll update the wireframes to show the entire end-to-end flow rather than the current one's non-linear states.

**Comment by chongho** [comment]
![q-box-v1-wireframes-r2](https://cloud.githubusercontent.com/assets/1796742/19623068/2cf905f2-9870-11e6-8cac-118358b1c34f.png)
